build/Tests/Benchmark/Framework.Benchmark.vcxproj

```diff
@@ -221,7 +221,9 @@
   </ItemGroup>
   <ItemGroup>
     <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\BenchmarkSampleTests.mm" />
+    <ClangCompile Include="$(StarboardBasePath)\tests\Benchmark\KVOBenchmarkTests.mm" />
```

Comment:
There is also a tests\testapps\kvoperf, which is probably obsolete now if we move all those tests here.